platforms/common/uORB/uORBDeviceNode.hpp

```diff
@@ -60,7 +60,7 @@ typedef void *uorb_cb_handle_t;
 #if defined(CONFIG_BUILD_KERNEL)
 #define MAX_EVENT_WAITERS 32
 #else
-#define MAX_EVENT_WAITERS 6
```

Comment:
This is ment for CONFIG_BUILD_PROTECTED only. This is a bit unclear, could be commented or re-organize the ifdefs